app/helpers/cmr_helper.rb

```diff
@@ -37,12 +36,13 @@ def flatten_to_array(collection_hash, parent_string = "")
           new_collection_arr.concat(flattened_sub_array)
         elsif sub_value.is_a?(Array)
           #some keyword groupings are presented as lists of strings, do not want to seperate these
-          if sub_value[0].is_a?(String)
+          if ( sub_value[0].is_a?(String) || sub_value[0].is_a?(Numeric))
```

Comment:
something change with CMR?

Reply:
maybe add a comment to the code of what changed

Reply:
Maybe array of numbers is new for this code.

Reply:
Just curious, what field was it parsing that had an array of numbers?